spec/models/spotlight/resource_spec.rb

```diff
@@ -49,6 +49,17 @@
         subject.reindex
       end
 
+      context "when the index is not writable" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.